tests/manual-test-cases/Group5-Functional-Tests/5-22-NFS-Volume.md

```diff
@@ -0,0 +1,83 @@
+Test 5-22 - NFS Volume
```

Comment:
This really looks good:
~1. Please add the new test into Test-Cases.md as well so it is indexed~
2. Is there any variations on the install itself related to nfs shares?  All your variations in here are just after VCH install...